Neos.SiteKickstarter/Resources/Private/Generator/Configuration/NodeTypes.Document.Page.yaml

```diff
@@ -2,7 +2,16 @@
 # to your pages. You can add as many areas as you like, still only those that are rendered in your Fusion-code and
 # Template are visible on your page.
 ---
-#'Neos.NodeTypes:Page':
+#'{packageKey}:Page':
 #  childNodes:
 #    'sidebar':
-#      type: 'Neos.Neos:ContentCollection'
\ No newline at end of file
+#      type: 'Neos.Neos:ContentCollection'
```

Comment:
I'd move this comment down under the actual node type, so that when the user uncomments the markup is actually valid. Like this:

```
'Foo.Bar:Page':
  superTypes:
    'Neos.Neos:Document': true
  ui:
    label: "Foo.Bar Custom Page"
    icon: 'icon-file'
  childNodes:
    main:
      type: 'Neos.Neos:ContentCollection'
      
#   If you need more than just the "main" content area you can uncomment the snippet below. It will add a "sidebar" area
#   to your pages. You can add as many areas as you like, still only those that are rendered in your Fusion-code and
#   Template are visible on your page.

#    sidebar:
#      type: 'Neos.Neos:ContentCollection'
```